packages/wrangler/templates/service-bindings-module-facade.js

```diff
@@ -28,7 +28,7 @@ export default {
 						}
 						const url = `${details.protocol}://${details.host}${
 							details.port ? `:${details.port}` : ""
-						}`;
+						}${details.pathname ? `${details.pathname}` : ""}`;
```

Comment:
This is still going to be missing any query params.

I'd re-write it to be like:

```
const url = new URL(req.url)
url.protocol = details.protocol
url.host = details.host
if (details.port) url.port = details.port
```